tests/container/build.sh

```diff
@@ -64,22 +144,12 @@ export WRITABLE_PASSWD=yes
 export NSS_WRAPPER_PASSWD=/etc/passwd
 export NSS_WRAPPER_GROUP=/etc/group
 
-# Run tox with sitepackages enabled to allow access to system installed samba
-# modules. The container env already provides us control over the env.
-info "running test suite with tox"
-tox
```

Comment:
On my machine, I get the following warn when when building:
```
py3-mypy run-test: commands[0] | mypy sambacc tests
WARNING: test command found but not installed in testenv
  cmd: /usr/bin/mypy
  env: /var/tmp/build/sambacc/.tox/py3-mypy
Maybe you forgot to specify a dependency? See also the allowlist_externals envconfig setting.

DEPRECATION WARNING: this will be an error in tox 4 and above!
```

(thou I could not figure out why)

Reply:
Yeah, I think the solution is not to have mypy in the system packages, but we need it for rpm build... there are other things I/we could look into for rpm building - like 'mock' but as it was only a warning I left it to be solved later.